config/default/manager_default_images.yaml

```diff
@@ -177,3 +177,5 @@ spec:
           value: quay.io/podified-antelope-centos9/openstack-ansible-tests:current-podified
         - name: RELATED_IMAGE_TEST_HORIZONTEST_IMAGE_URL_DEFAULT
           value: quay.io/podified-antelope-centos9/openstack-horizontest:current-podified
+        - name: RELEATED_IMAGE_OPENSTACK_MUST_GATHER_DEFAULT
```

Comment:
As per the interface defined in the commit [1], it seems the right place to define the `must-gather` image. We can solve the problem of disconnected environments and provide documentation about this particular use case.
I'm ok with this change if it represents the best practice in the `operator-framework` context.


[1] https://github.com/operator-framework/api/commit/4f51a5d0bab048fa7d3751396ebef662cf8b69a5#diff-3490b8e064835cb2059f50d7a4adda830a7a048708976c257f04ca6de42b9742R317